.coveragerc

```diff
@@ -0,0 +1,15 @@
+[run]
+branch = True
+source = z3c.recipe.mkdir
+
+[report]
+precision = 2
+
+[html]
+directory = htmlcov
+
+[paths]
+source =
+   z3c/recipe/mkdir
+   .tox/*/lib/python*/site-packages/z3c/recipe/mkdir
+   .tox/pypy*/site-packages/z3c/recipe/mkdir
```

Comment:
I don't think this is necessary?  You only need this when you use `coverage combine` to combine the coverages of several runs (that tracked the source files in multiple directories).

OTOH it doesn't hurt either, and maybe somebody someday will want to use `coverage combine`.  *shrug*

Reply:
I kept it that way.